hw2/05-form.js

```diff
@@ -0,0 +1,30 @@
+const form = document.querySelector('form');
+
+function handleSubmission(evnt) {
```

Comment:
you use e in the other file, and evnt here; might be better to use consistent naming throughout

Reply:
I agree, I'll make this change.